### DISCUSSION TITLE
wrong lock file version being generated by dependabot for pnpm.

### DISCUSSION THREAD

**Discussion by Harsh-Sharma25** [dependabot/dependabot-core#10138] [Q&A] [answered]
My package manager is pnpm with version 8.15.1.
This is also specified in my package.json.
But for some reason dependabot is rewriting the whole lock file in v9 in one of my repo.

`lockfileVersion: '6.0'`
gets changed into 
`lockfileVersion: '9.0'` 
along with whole lock file being re written according to version 9 which causes breaking of lock-file.

Unfortunately my repo isn't public and this started happening on version updates that were opened by dependabot like 2 weeks before.
And since then all the PR's opened by dependabot have the same issue.
I don't know how to reproduce it as in my other repos, this issue hasn't appeared. 

I've re-created & rebased the PR but still it rewrites it in v9.

**Comment by abdulapopoola** [comment, marked as answer]
Hi @Harsh-Sharma25 , it does seem this might be resolved by #10073 ; please let us know if it still happens.

**Reply by Harsh-Sharma25** [reply]
Yeah doesn't happening after recreate or on newly created PR's.

**Reply by abdulapopoola** [reply]
Thanks for confirming, will close the discussion.
